projects/wmf_default.yaml

```diff
@@ -141,9 +138,10 @@ paths:
             /mobileapps:
               x-modules:
                 - path: sys/mobileapps.yaml
-                  options: '{{options.mobileapps}}'
+                  options: '{{merge({"response_cache_control": options.purged_cache_control}, 
```

Comment:
Why do we allow overriding it for mobile apps and not for parsoid and others? Lets be consistent.